schematic/visualization/tangled_tree.py

```diff
@@ -258,11 +277,14 @@ def _get_ca_alias(self, conditional_requirements: list[str]) -> dict[str, str]:
                 value: attribute
         """
         ca_alias: dict[str, str] = {}
+        extracted_conditional_requirements = self._extract_content_within_quotes(
```

Comment:
Problem this is addressing:
```
  File "/home/bfauble/BryansGreatWorkspace/schematic/schematic/visualization/tangled_tree.py", line 292, in _get_ca_alias
    attr, ali = req.split(" is ")
ValueError: too many values to unpack (expected 2)
```

I was seeing some test failures pop up in the logic `attr, ali = req.split(" is ")` below.

The reason is that in an integration test `.split` was splitting and returning back more than 2 values on this input:

- `['[\'File Format is "CRAM"\', \'File Format is "BAM"\', \'File Format is "CSV/TSV"\']', '[\'File Format is "CRAM"\']']`

**There are a few problems for this put:**

1. It contained multiple strings of `" is "`, causing the immediate exception to be thrown.
2. Nothing in this code was properly formatting the content out. The result return statement of this function was:

**After this change, the output of the function is:**

Input: `['[\'Diagnosis is "Cancer"\']', '[\'Diagnosis is "Cancer"\']']`
Output: `{'Cancer': 'Diagnosis'}`

Input: `['[\'File Format is "CSV/TSV"\', \'File Format is "BAM"\', \'File Format is "CRAM"\']', '[\'File Format is "CRAM"\']']`
Output: `{'CSV/TSV': 'FileFormat', 'BAM': 'FileFormat', 'CRAM': 'FileFormat'}`


**Example of malformed content:**
Input: `['[\'Diagnosis is "Cancer"\']', '[\'Diagnosis is "Cancer"\']']`
Output: `{'"Cancer"\']': "['Diagnosis"}`